pkgs/applications/networking/browsers/floorp/default.nix

```diff
@@ -38,7 +38,7 @@
     # thus is the full /nix/store/[..] path. To avoid breaking PWAs with each
     # update, rely on `floorp` being in $PATH, as before.
     substituteInPlace floorp/browser/base/content/modules/ssb/LinuxSupport.mjs \
-      --replace-fail 'Services.dirsvc.get("XREExeF",Ci.nsIFile).path' floorp
+      --replace-fail 'Services.dirsvc.get("XREExeF",Ci.nsIFile).path' 'floorp'
```

Comment:
If you want the quotes, you need to quote them. Otherwise bash eats them.

Reply:
Just realized that, thank you! The compile just takes really long.